sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/AsynchronousMetricStorage.java

```diff
@@ -45,13 +51,15 @@
     final MetricDescriptor metricDescriptor = MetricDescriptor.create(view, instrument);
     Aggregator<T> aggregator =
         view.getAggregation()
-            .config(instrument)
-            .create(
+            .createAggregator(
                 resource,
                 instrumentationLibraryInfo,
                 instrument,
                 metricDescriptor,
-                ExemplarReservoir::noSamples);
+                ExemplarFilter.neverSample());
+    if (Aggregator.empty().equals(aggregator)) {
```

Comment:
== here as well?

Reply:
DOH! Thought I had caught that, my bad. Fixed.